packages/atomic/cypress/integration/search-interface-utils.cypress.ts

```diff
@@ -0,0 +1,36 @@
+import {i18n} from 'i18next';
```

Comment:
Most of these functions and interfaces were extracted from `search-interface.cypress.ts` below.